src/modules/add.js

```diff
@@ -0,0 +1,29 @@
+import listStore from './listStore.js';
+import displayList from './displayList.js';
+import retrieveFromStore from './retrieveFromStore.js';
+
+// Add a todo item to the store;
+const add = () => {
+  function Todo(index, description, isCompleted = false) {
+    this.index = index;
+    this.description = description;
+    this.isCompleted = isCompleted;
+  }
+
+  const inputField = document.querySelector('.todo-input');
+
+  inputField.addEventListener('keyup', (event) => {
+    if (event.key === 'Enter') {
+      const listArray = retrieveFromStore();
+      const inputValue = inputField.value;
+      const index = listArray.length;
```

Comment:
By default, newly added tasks possess the property `completed` set to false, which is a good start. However, there's a slight adjustment required:

- [ ] According to the project specifications, new tasks should also be equipped with the property `index`, set to the value of the new array length. Presently, if the third task is appended to the list, its index should be 3. However, in your current implementation, the index value of the 3rd task is 2. I recommend revising your code to align with the project's requirements and ensure accurate indexing.
![image](https://github.com/ismayito/Todo-list-project/assets/55840999/6743106c-03e6-4d75-a657-da7f8702700c)